### DISCUSSION TITLE
Starting Directory being ignored

### DISCUSSION THREAD

**Discussion by HiDannyZ** [microsoft/WSL#10394] [General]
### Windows Version

Microsoft Windows [Version 10.0.22621.2134]

### WSL Version

WSL version: 1.2.5.0

### Are you using WSL 1 or WSL 2?

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.15.90

### Repro Steps

I am trying to set my starting directory to a folder on my Desktop.

These are the steps I am following and I have been trying for the last 40 mins following guides online.

1. Open Ubuntu Profile on windows terminal via UI or settings.json file
2. Set Starting Directory to ```/c/Users/Danny/Desktop/Projects``` or ```/mnt/c/Users/Danny/Desktop/Projects```
``` 
            {
                "elevate": false,
                "guid": "{51855cb2-8cce-5362-8f54-464b92b32386}",
                "hidden": false,
                "name": "Ubuntu",
                "source": "CanonicalGroupLimited.Ubuntu_79rhkp1fndgsc",
                "startingDirectory": "/mnt/c/Users/Danny/Desktop/Projects"
            },
```

### Expected Behavior

Windows terminal Ubuntu will now open in the projects folder on my desktop

### Actual Behavior

Windows Terminal Ubuntu ignores all changes that I make to the starting directory.

No matter the change I always start at ```/home/hidanny```.

Only when the starting directory is invalid, there is an effect via error on the ubuntu instance in windows terminal.

**Comment by benhillis** [comment]
I believe this issue should be posted in the Windows Terminal repo.
